deploy-service/common/src/main/java/com/pinterest/deployservice/handler/PingHandler.java

```diff
@@ -154,7 +155,8 @@ void updateHosts(String hostName, String hostIp, String hostId, Set<String> grou
                 groupsToAdd.add(group);
             }
         }
-        if (groupsToAdd.size() > 0) {
+        
+        if (groupsToAdd.size() > 0 || accountId != null && !accountId.equals(recordedAccountId)) {
```

Comment:
let's update it for sub account for now to avoid redundant updates.

Reply:
updated, thanks